app/src/components/keyCloakServiceClientMgr.js

```diff
@@ -169,17 +169,23 @@ class KeyCloakServiceClientManager {
     }
   }
 
-  async fetchAllClients() {
-    log.info('KeyCloakServiceClientManager.fetchAllClients');
+  async fetchAllClients(kcEnv) {
+
+    log.info('KeyCloakServiceClientManager.fetchAllClients in ' + kcEnv);
 
     //get all service clients
-    // NOTE: this seems to return all the service clinets but shows a ERR 400 in the console for all the methods in the reamlAdminSvc class
-    const clients = await this.svc.getClients();
+    let clients = await this.svc.getClients();
```

Comment:
It's not great practice to be reassigning something you get from a response. It's better to either create another const variable and manipulate, or consider function chaining to remove the need altogether.